pyproject.toml

```diff
@@ -27,6 +27,11 @@ dev = [
     "seaborn",
 ]
 
+test = [
```

Comment:
this is not good. these two dependencies seem very heavy. is there an alternative?

Reply:
these won't be installed by default thought. only when you `pip install . '[test]'` they are installed. 

they are not heavy actually. installation takes about 15 seconds